preprocessors/openstreetmap/main.py

```diff
@@ -54,26 +65,58 @@ def get_map_data():
 
     if validated is not None:
         return validated
-
+    time_stamp = int(get_timestamp())
+    name = "ca.mcgill.a11y.image.preprocessor.openstreetmap"
+    request_uuid = content["request_uuid"]
     # Check if this request is for an openstreetmap
     if 'coordinates' not in content and 'placeID' not in content:
-        logging.info("Not map content. Skipping...")
-        return "", 204
+        LOGGER.info("Not map content. Skipping...")
```

Comment:
Great warning message! But you aren't exactly skipping are you? I don't know if the policy changed, but you are still sending a response for JSON for this preprocessor. The response has no data in it so you are skipping in a sense, but as far as I see for other preprocessors, when you skip you don't return a JSON. No point in populating the output with empty data! Please correct me if I am wrong or if there has been a change in how to go about this!